Dockerfile

```diff
@@ -13,6 +13,7 @@ COPY . /go/src/github.com/openshift/origin
 ENV GOPATH=/go \
     PATH=$PATH:$GOROOT/bin:$GOPATH/bin
 
+RUN getent hosts mirrorlist.centos.org
```

Comment:
don't we want a single RUN statement for all of these to avoid adding another layer? I would expect this command `&&`-ed with the other commands